ansible_events/drools/durable_rules_engine.py

```diff
@@ -0,0 +1,230 @@
+import json
+import logging
+
+import requests
+
+_logger = logging.getLogger(__name__)
+
+
+class DurableRulesEngine:
+    __host = None
+    __last_resp = None
+
+    def __init__(self, host):
+        self.__host = host
+
+    def create_ruleset(self, ruleset_name, ruleset_string):
+        req = {ruleset_name: json.loads(ruleset_string)}
+
+        r = requests.post(
```

Comment:
It seems like single character names are heavily overused here and below. Could you use more verbose names, like:

* `r -> response`
* `d -> data`

etc.?

Reply:
sure